actionpack/lib/abstract_controller/caching/fragments.rb

```diff
@@ -135,6 +135,7 @@ def expire_fragment(key, options = nil)
       end
 
       def instrument_fragment_cache(name, key) # :nodoc:
+        return yield unless enable_fragment_cache_logging
```

Comment:
Ok, I think I don't very understand the differences between `instrumentation` and `logging`. I thought they are the same thing 😓